easybuild/easyconfigs/g/GATE/GATE-8.2_fix_std_max.patch

```diff
@@ -0,0 +1,12 @@
+diff -Nru Gate-8.2.orig/source/general/src/GateIAEAPhsp.cc Gate-8.2/source/general/src/GateIAEAPhsp.cc
```

Comment:
Please add a description and author information to the patch

Reply:
done